### PR TITLE
Remove lured pokemon toggle, fix pokestop with no lured pokemon

### PR DESCRIPTION
**Pull request by invisiblek**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

* Base showing lured pokemon based on whether or not we show any at all

### CODE REVIEW

File: static/map.js [hunk 1]
```diff
@@ -638,7 +637,7 @@ function processPokestops(i, item) {
 }
 
 function processLuredPokemon(i, item) {
```
Comment (efiku) [inline static/map.js]: If you are going to remove broken functionality please remove unused functions too.
For Web Development i recommend WebStorm / PHPStorm.

Comment (invisiblek) [inline static/map.js]: a) I will never use an IDE
b) This function is still needed as lured pokemon need to be processed differently than regular ones. We still want to show them, we just want to show them based on the "Show Pokemon" toggle now.

Comment (efiku) [inline static/map.js]: +1 :)

**Comment by invisiblek** [comment]
going to need this: https://github.com/AHAAAAAAA/PokemonGo-Map/pull/1994  also

**Comment by hgentry** [comment]
There are several PRs dealing with lures right now, fixing various issues. It seems like they keep getting fixed and re-broken every few commits because people don't realize we have to load pokestop data from the db in order to display them. #1829 #1849 #1982 #1994

**Comment by MonkeyLeeT** [comment]
@invisiblek Can we spend one PR to make a decision on lured pokestops? There're three milestone blockers related to lured pokes tops now...

**Comment by JZ6** [comment]
close #1952 when merging

File: static/map.js [hunk 2]
```diff
@@ -779,6 +778,8 @@ function processLuredPokemon(i, item) {
         disappear_time: item.lure_expiration
     };
 
+    if (item2.pokemon_id == null){
+        return;
```
Comment (parkershepherd) [inline static/map.js]: Missing `}` breaks Chrome (but not Firefox for some reason)

Comment (invisiblek) [inline static/map.js]: oh wtf

**Pull request merged**